ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -52,6 +57,23 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
       this.can.can('connect target', target),
     );
 
-    return { targets: targets, totalItems };
+    let targetsWithSessions = [];
+    for (const target of targets) {
+      let sessions = await this.store.query('session', {
+        query: {
+          filters: {
+            target_id: { equals: target.id },
+            status: [
+              { equals: STATUS_SESSION_ACTIVE },
```

Comment:
Does `equals` only take a string? wondering if it can take in an array so we can combine these statuses in an array 
```suggestion
              { equals: STATUS_SESSION_ACTIVE },
```